src/components/NewClusterModal.js

```diff
@@ -1103,6 +1107,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
           onPrevious: () => this.setState({ viewMode: undefined })
         }),
         div({ style: { lineHeight: 1.5 } }, [
+          p(['Your persistent disk is mounted in the directory ', code({ style: { fontWeight: 600 } }, ['/home/jupyter-user/notebooks']), '. Please save your analysis data in this directory to ensure it’s stored on your disk.']),
```

Comment:
in the mocks the `/home/jupyter-user/notebooks` got quotes and a new line, should that be the case here too?

Reply:
I added the line break. The quotes were copied directly from the Adrian's original proposal, but I think the monospace code font and font-weight do a better job of delineating this (and also making copy-pasting less error prone).